lib/server/readMetadata.js

```diff
@@ -122,15 +122,18 @@ function extractMetadata(content) {
 }
 
 // process the metadata for a document found in the docs folder
-function processMetadata(file) {
+function processMetadata(file, _language) {
   const result = extractMetadata(fs.readFileSync(file, 'utf8'));
 
-  let regexSubFolder = new RegExp(
-    '/' + escapeStringRegexp(getDocsPath()) + '/(.*)/.*/'
-  );
+  let language = _language;
```

Comment:
why don't we delete this line and replace `_language` at line 125 to be `language`
Save one line of code ? 😃

Reply:
This 3 lines is moved to the next line.

Reply:
I mean
```js
function processMetadata(file, language) {
```

instead of 
```js
function processMetadata(file, _language) {
let language = _language;
```